plugin-build/src/test/kotlin/io/sentry/android/gradle/BaseSentryPluginTest.kt

```diff
@@ -99,10 +99,14 @@ abstract class BaseSentryPluginTest(
             arrayOf("7.0.4", "7.2"),
             arrayOf("7.1.2", "7.2"),
             arrayOf("7.1.2", "7.3.3"),
-            arrayOf("7.1.3", "7.4"),
-            arrayOf("7.2.0", "7.4"),
-            arrayOf("7.3.0-beta01", "7.4"),
-            arrayOf("7.4.0-alpha01", "7.4.2")
+            arrayOf("7.1.3", "7.2"),
+            arrayOf("7.1.3", "7.5"),
```

Comment:
We don't need to test with every patch from AGP.
Just change `7.1.2` to `7.1.3` and add a new case for Gradle 7.4 and 7.5

Reply:
updated